arviz/plots/backends/matplotlib/dotplot.py

```diff
@@ -0,0 +1,86 @@
+import matplotlib.pyplot as plt
+
+from ...plot_utils import _scale_fig_size
+from . import backend_kwarg_defaults, create_axes_grid
+from .plotpointinterval import plot_point_interval
+
+
+def plot_dots(
+    values,
+    binwidth,
+    dotsize,
+    stackratio,
+    hdi_prob,
+    rotated,
+    dotcolor,
+    intervalcolor,
+    markersize,
+    figsize,
+    linewidth,
+    point_estimate,
+    quantiles,
+    point_interval,
+    backend_kwargs,
+    plot_kwargs,
+    interval_kwargs,
+):
+
+    if backend_kwargs is None:
+        backend_kwargs = {}
+        backend_kwargs = {
+            **backend_kwarg_defaults(),
+        }
+
+        backend_kwargs.setdefault("figsize", figsize)
+        backend_kwargs["squeeze"] = True
+
+    (figsize, _, _, _, auto_linewidth, auto_markersize) = _scale_fig_size(figsize, None)
+
+    if plot_kwargs is None:
+        plot_kwargs = {}
+        plot_kwargs.setdefault("color", dotcolor)
+
+    if linewidth is None:
+        linewidth = auto_linewidth
+
+    if markersize is None:
+        markersize = auto_markersize
+
+    if interval_kwargs is None:
+        interval_kwargs = {}
+        interval_kwargs.setdefault("color", intervalcolor)
+
+    _, ax = create_axes_grid(1, backend_kwargs=backend_kwargs,)
+
+    ## Wilkinson's Algorithm
```

Comment:
For modularity, could you split this out into its own function?

Reply:
Sure